pkg/chain/celo/tbtc.go

```diff
@@ -0,0 +1,462 @@
+//+build celo
+
+package celo
+
+import (
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/celo-org/celo-blockchain/common"
+	"github.com/keep-network/keep-common/pkg/chain/celo/celoutil"
+	"github.com/keep-network/keep-common/pkg/subscription"
+	"github.com/keep-network/keep-ecdsa/pkg/chain"
+	"github.com/keep-network/keep-ecdsa/pkg/chain/gen/celo/contract"
+	tbtcchain "github.com/keep-network/tbtc/pkg/chain/celo/gen/contract"
+)
+
+// tbtcApplication represents an Celo chain handle with
```

Comment:
I forgot to update this on Ethereum as well, will adjust both to clarify a little in 714 post-merge.